docs/source/basicusage.rst

```diff
@@ -123,6 +137,7 @@ If you want an external program to access your API server, an open firewall port
      - 10332 
      - 20332
 
+For P2P and WebSocket information see `NEO-Python Seedlist <https://neo-python.readthedocs.io/en/latest/Seedlist.html>`_.
```

Comment:
neo-python does not use or support websockets. Can you please remove mentioning `websockets` as it might raise questions or false expectations

Reply:
done.